phpcs.xml.dist

```diff
@@ -7,6 +7,8 @@
 
 	<rule ref="WordPress">
 		<exclude name="WordPress.XSS.EscapeOutput"/>
+		<exclude name="WordPress.Arrays.ArrayIndentation"/>
```

Comment:
I need to remove these two lines now that @westonruter has fixed up the CI problems.